warehouse/slave.go

```diff
@@ -197,10 +197,14 @@ func (jobRun *JobRunT) uploadLoadFilesToObjectStorage() ([]loadFileUploadOutputT
 						uploadErrorChan <- err
 						return
 					}
+					contentLength := loadFileStats.Size()
+					if contentLength == 0 {
+						panic(fmt.Errorf("[WH]: Empty load file generated in slave for tablename: %v", uploadJob.tableName))
```

Comment:
We can also add the uploadId to the log, this would help us in identifying the affected uploads.

Reply:
Done.